src/openzeppelin/utils/constants.cairo

```diff
@@ -36,3 +36,27 @@ const IACCESSCONTROL_ID: u32 = 0x7965db0b_u32;
 //
 
 const DEFAULT_ADMIN_ROLE: felt252 = 0;
+
+//
+// Selectors
+//
+
+const HAS_ROLE_SELECTOR: felt252 =
+    0x30559321b47d576b645ed7bd24089943dd5fd3a359ecdd6fa8f05c1bab67d6b;
+const HASROLE_SELECTOR: felt252 = 0x35ed3407ba17dc741dd3af821fa1548619ebcdc87c95bcea9e3bc510951fae8;
+const GET_ROLE_ADMIN_SELECTOR: felt252 =
+    0x302e0454f48778e0ca3a2e714a289c4e8d8e03d614b370130abb1a524a47f22;
+const GETROLEADMIN_SELECTOR: felt252 =
+    0x2034da88f3e3f3382ab0faf97fe5f74fe34d551ddff7fda974d756bf12130a0;
+const GRANT_ROLE_SELECTOR: felt252 =
+    0x18a2f881894a5eb15a2a00f598839abaa75bd7f1fea1a37e42779d7fbcd9cf8;
+const GRANTROLE_SELECTOR: felt252 =
+    0x37322ff1aabefe50aec25a14eb84b168b7be4f2d66fbbdb5dd8135e8234c37a;
+const REVOKE_ROLE_SELECTOR: felt252 =
+    0x246116ed358bad337e64a4df51cb57a40929189494ad5905a39872c489136ec;
+const REVOKEROLE_SELECTOR: felt252 =
+    0xa7ef1739dec1e216a0ba2987650983a3104c707ad0831a30184a3b1382dd7d;
+const RENOUNCE_ROLE_SELECTOR: felt252 =
+    0xd80093a4ee6a9e649f2ae3c64963d5096948d50cf4ea055500aa03a342fd43;
```

Comment:
we should update with the new constant name format for selectors as discussed in https://github.com/OpenZeppelin/cairo-contracts/pull/625

Reply:
Done!